data/TRohit20/events/2023-06-13-DevRelCon-London.json

```diff
@@ -9,5 +9,13 @@
       "end": "2023-06-14",
       "cfpClose": "2023-03-31"
     },
-    "url": "https://london-2023.devrelcon.dev"
+    "url": "https://london-2023.devrelcon.dev",
+    "location": {
+      "road": "Mariana Blvd",
+      "city": "San Francisco",
+      "state": "California",
+      "country": "United States",
+      "lat":37.774929,
+      "lon":-122.419418
+    }
```

Comment:
The event is not in SF, but London 

Any reason for making this change to this particular event?

Reply:
Sorry for that.... I was just trying out to put different locations...will change.